src/NuGet.Core/NuGet.CommandLine.XPlat/Commands/PackageReferenceCommands/ListPackage/ListPackageCommand.cs

```diff
@@ -85,10 +88,36 @@ public static class ListPackageCommand
                     Strings.NuGetXplatCommand_Interactive,
                     CommandOptionType.NoValue);
 
+                var verbosity = listpkg.Option(
+                    "-v|--verbosity",
+                    Strings.Verbosity_Description,
+                    CommandOptionType.SingleValue);
+
                 listpkg.OnExecute(async () =>
                 {
                     var logger = getLogger();
 
+                    switch (verbosity.Value()?.ToUpperInvariant())
+                    {
+                        case "Q":
```

Comment:
only thing I'd suggest is extracting it to a separate utility, but maybe that's premature at this point given that there are no other usages.

Reply:
Done.